recipes/r-lamw/meta.yaml

```
@@ -0,0 +1,82 @@
+{% set version = '1.3.3' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-lamw
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/lamW_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/lamW/lamW_{{ version }}.tar.gz
+  sha256: 74ddf5df3446d6329b22d91186bc4a7f8703232fc350dc8a9b9cde72c8f1e58a
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-rcpp
+    - r-rcppparallel >=4.3.20
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-rcpp
+    - r-rcppparallel >=4.3.20
+
+test:
+  commands:
+    - $R -e "library('lamW')"           # [not win]
+    - "\"%R%\" -e \"library('lamW')\""  # [win]
+
+about:
+  home: https://github.com/aadler/lamW
+  license: BSD_2_clause
```

```suggestion
  license: BSD-2-Clause
```

see spdx short: https://spdx.org/licenses/BSD-2-Clause.html

---

Thx